cpp/src/cider/util/CiderBitUtils.h

```diff
@@ -240,6 +239,20 @@ inline void ENABLE_AVX256 bitwiseAnd(uint8_t* __restrict output,
     output[i] = a[i] & b[i];
   }
 }
+
+inline void ENABLE_AVX256 byteToBit(uint8_t* __restrict byte,
+                                    uint8_t* __restrict bit,
+                                    size_t bit_num) {
+  __m256i* byte_256 = (__m256i*)byte;
+  int* bit_32 = (int*)bit;
+  bit_num += 31;
+  bit_num >>= 5;
+  for (size_t i = 0; i < bit_num; ++i) {
```

Comment:
can't normal loop auto-vectorization by the compiler?

Reply:
Yes, the compiler can't handle the vectorization of bit operations.